core/src/main/resources/filodb-defaults.conf

```diff
@@ -245,6 +245,25 @@ filodb {
       downsamplers = ["tTime(0)", "dSum(1)", "dMin(2)", "dSum(3)", "dMax(4)"]
       downsample-schema = "preagg-gauge"
     }
+
+    # new preagg schema to support last-over-time aggregations
+    preagg-gauge-lot {
+      columns = [
+        "timestamp:ts",
+        "count:double:detectDrops=false",
+        "min:double:detectDrops=false",
+        "sum:double:detectDrops=false",
+        "max:double:detectDrops=false",
+        "countl:double:detectDrops=false",
+        "minl:double:detectDrops=false",
+        "suml:double:detectDrops=false",
+        "maxl:double:detectDrops=false",
+      ]
+      value-column = "sum"
+      downsample-period-marker = "time(0)"
+      downsamplers = ["tTime(0)", "dSum(1)", "dMin(2)", "dSum(3)", "dMax(4)","dSum(5)", "dMin(6)", "dSum(7)", "dMax(8)"]
```

Comment:
@sherali42  the minl, maxl etc columns may need more thinking on downsampling. The `dLast` downsampler may be more apt for these.

Reply:
make sense. done 👍